epoch/api/contract_api_usage.md

```diff
@@ -0,0 +1,45 @@
+[back](./README.md)
+# Contracts - intended usage
```

Comment:
It would be nice mentioning in this intended usage doc a sample contract - or versioned link to working sample contract.